app/kaidan.pro

```diff
@@ -1,5 +1,5 @@
 # path to local compiled swift 3 lib
-SWIFT3PATH = ../../../swift-3.0
+SWIFT3PATH = ../../../swift-swift-3.0
```

Comment:
Either I don't understand something or "Kaidan" directory and "swift-3.0" directory are within "src" directory, that means it should be "../../swift-3.0"